### ISSUE TITLE
UNVT Portable Bazaar experiment: porting the bazaar to Prizren

### ISSUE THREAD

**Issue by hfu** [UNopenGIS/7#195]
@hfu is trying to host Smart Maps Bazaar in ITP Prizren by locating UNVT Portable Bazaar there. 

## Control experiment（対照実験）: without Prizren node
@yuiseki, @smellman could you help by checking if the following links works from Prizren (without my server in ITP Prizren.) Cc: @mapconcierge, @ShogoHirasawa. 

My guess is that the following links does not work or works really slow because the nodes are too far away.

I appreciate if you could run the following beforehand for peering. This will connect the IPFS node hosted at smb.optgeo.org, in-kind cooperation by Code for Japan and Sakura Internet. smb.optgeo.org does not pin anything, but my nodes keep connection with smb.optgeo.org. 

```
curl --silent https://unopengis.github.io/smb/connect.sh | sh
```

### Open Nagasaki Point Cloud
- `ipfs://bafybeif7pedxbcrdgxyjd5rbihljehdybf7kjncugfbmh52hinqvsflpg4/index.html?tileset=ipfs://bafybeif7pedxbcrdgxyjd5rbihljehdybf7kjncugfbmh52hinqvsflpg4/tileset/tileset.json#32.748800/129.879511/160/21/-37`
- https://smb.optgeo.org/ipfs/bafybeif7pedxbcrdgxyjd5rbihljehdybf7kjncugfbmh52hinqvsflpg4/index.html?tileset=https://smb.optgeo.org/ipfs/bafybeif7pedxbcrdgxyjd5rbihljehdybf7kjncugfbmh52hinqvsflpg4/tileset/tileset.json#32.748800/129.879511/160/21/-37

If successful, this site should look like the following: 

<img width="1510" alt="スクリーンショット 2023-06-24 15 31 46" src="https://github.com/UNopenGIS/7/assets/18297/fb8abb02-16d0-40d5-ac45-dd176597ed3c">

### Plateau 3D Tiles (Chuo-ku)
- `ipfs://bafybeicmms2ggu2oyllgfgti2br75iosqcmiefgokn7rudhfhcd3u4dst4/o.html?cid=QmUcFnzK2XtEnd5dpRdypemUCctWycn6Afxa526NQDFH5d#35.671122/139.775671/283/5/-37`
- https://smb.optgeo.org/ipfs/bafybeicmms2ggu2oyllgfgti2br75iosqcmiefgokn7rudhfhcd3u4dst4/o.html?cid=QmUcFnzK2XtEnd5dpRdypemUCctWycn6Afxa526NQDFH5d#35.671122/139.775671/283/5/-37

If successful, this site should look like the following: 

<img width="1508" alt="スクリーンショット 2023-06-24 15 31 32" src="https://github.com/UNopenGIS/7/assets/18297/46d6e666-99a6-440b-91a6-afad958d25f9">

### 地図XML (ab: amx-a on bazaar)
- `ipfs://bafybeihtpslpnptigzlpyvdxnvojnrg5adjhlqaa2sylipx4cfhyfkdvdi/#17.67/38.234279/140.862063`
- https://smb.optgeo.org/ipfs/bafybeihtpslpnptigzlpyvdxnvojnrg5adjhlqaa2sylipx4cfhyfkdvdi/#17.67/38.234279/140.862063

<img width="1509" alt="スクリーンショット 2023-06-24 15 34 02" src="https://github.com/UNopenGIS/7/assets/18297/fe119ca2-5bc9-4188-aaf3-7dd3e6ea9382">

**Comment by hfu** [comment]
Arrived in ITP. `IPKO-FOSS4G` WiFi network is good!

<img width="968" alt="スクリーンショット 2023-06-25 14 26 10" src="https://github.com/UNopenGIS/7/assets/18297/2458c8c4-1319-4d17-b55d-1bbebd33860d">

**Comment by hfu** [comment]
Successfully implemented as #196. Thanks!
